tests/unit/lib/build_module/test_app_builder.py

```diff
@@ -244,6 +246,67 @@ def test_must_skip_if_no_artifacts(self):
         self.assertEqual(actual, self.template_dict)
 
 
+class TestApplicationBuilder_update_template_windows(TestCase):
```

Comment:
Should this have a `@skipIf(IS_WINDOWS)` so it only runs in windows?

Reply:
It only runs on Windows previously, but since it's unit test, I have changed it to be runnable on other systems.